tools/mount_gcsfuse/main.go

```diff
@@ -121,7 +124,8 @@ func makeGcsfuseArgs(
 			"debug_gcs",
 			"debug_http",
 			"debug_invariants",
-			"debug_mutex":
+			"debug_mutex",
+			"foreground":
```

Comment:
putting foreground in this case because, when I passed foreground=false in argument, VM was crashing.

Reply:
As discussed offline, we should look into why VM is crashing when foreground is put in the above case.

Reply:
checked on another VM. It was working fine. May be it was VM issue